test/integration/features/cmd-hostfolder.feature

```diff
@@ -0,0 +1,104 @@
+@cmd-hostfolder @commands
```

Comment:
@LalatenduMohanty Can you please add `@core` here to add the file to PR testing set?

Reply:
@LalatenduMohanty ping ...